app/components/CommentInput/CommentInput.jsx

```diff
@@ -167,12 +168,14 @@ CommentInput.propTypes = {
   questionId: PropTypes.number,
   isWritingCommentMobile: PropTypes.bool,
   setWritingCommentOnMobile: PropTypes.func,
+  is_public: PropTypes.bool.isRequired,
```

Comment:
Here you can delete the isRequiered

```suggestion
  is_public: PropTypes.bool,
```